Doc/library/doctest.rst

```diff
@@ -300,33 +301,49 @@ their contained methods and nested classes.
 How are Docstring Examples Recognized?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The :mod:`doctest` module treats any line beginning with ``>>>`` as an 
+example to be tested. 
+Following lines which begin with ``...`` continue the example. 
+Subsequent non-blank lines, if any, form an expected output string. 
+A blank (all-whitespace) line, or a line with ``>>>`` (beginning the 
+next example), ends the expected output string.
+
 In most cases a copy-and-paste of an interactive console session works fine,
 but doctest isn't trying to do an exact emulation of any specific Python shell.
 
 ::
 
    >>> # comments are ignored
-   >>> x = 12
-   >>> x
-   12
-   >>> if x == 13:
-   ...     print("yes")
+   ... 
```

Comment:
The bug being, the Python 3.7 in-dev interactive shell receives a comment-only line, and prints a sys.ps2 prompt `...` in response. A sys.ps1 `>>>` prompt is expected.  Yes, this behaviour seemed strange. Python2.7 behaves as expected. I'm happy to delete the `...` line after the comment line.  Should I also file the bug report, or do you want to?